js/stringProcessing/removePunctuation.js

```diff
@@ -1,18 +1,18 @@
-// Replace all other punctuation chars at the beginning or at the end of a word.
-var punctuationRegexString = "[\-()_\\[\\]’“”\"'\/]";
+// Replace all other punctuation characters at the beginning or at the end of a word.
+var punctuationRegexString = "[\\–\\-\\(\\)_\\[\\]’“”\"'.?!:;,¿¡«»]+";
 var punctuationRegexStart = new RegExp( "^" + punctuationRegexString );
 var punctuationRegexEnd = new RegExp( punctuationRegexString + "$" );
 
 /**
- * Replaces punctuation chars from the given text string.
+ * Replaces punctuation characters from the given word string.
  *
- * @param {String} text The text to remove the punctuation chars for.
```

Comment:
Does this function require a word to be input to work? I think `text` is correct in the documentation.